build-all.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
This is quite cool. Not a direct comment on this one, but just a thought: Would be nice to have a `util` directory with all of the shell scripts we have strewn around the project. The ErgoDox EZ has a `makeallhex.sh` for example, which does something similar but not quite like this.